server.py

```diff
@@ -180,5 +189,8 @@ def test_sql(connection):
 if __name__ == '__main__':
     # Startup
     app.logger.info("Starting server: version='{}'".format(__version__))
-    port = int(os.getenv("PORT"))
-    app.run(debug=True, host='0.0.0.0', port=port)
+    if os.getenv("CF_DEPLOYMENT"):
+        app.run(debug=True)
+    else:
+        port = int(os.getenv("PORT"))
+        app.run(debug=True, host='0.0.0.0', port=port)
```

Comment:
Do we want to have `debug=True` here?

Reply:
I think I need to pick this up with Warren . It looks to be that some services , mostly the front end of SDX have a SDX_DEV_MODE in which case they used to run with debug = true now we only run in Dev mode for non cloud foundry deploys . This may need to be consistent across all services

Reply:
Yes this should be consistent, though this main method is only used locally. It shouldn't be used in preprod/prod or cloudfoundry